htsget-http-actix/src/main.rs

```diff
@@ -1,59 +1,21 @@
 use std::env::args;
 use std::sync::Arc;
 
-use crate::handlers::blocking::{get, post, reads_service_info, variants_service_info};
+//use htsget_http_actix::handlers::blocking::{get, post, reads_service_info, variants_service_info};
```

Comment:
I think this shouldn't be commented, but rather have `#[cfg(not(feature = "async"))]`, so it only gets used in the blocking version